dev/buildchecker/main.go

```diff
@@ -342,7 +348,45 @@ func cmdHistory(ctx context.Context, flags *Flags, historyFlags *cmdHistoryFlags
 		}
 	}
 
+	if historyFlags.okayHQToken != "" {
+		okayCli := okay.NewClient(http.DefaultClient, historyFlags.okayHQToken)
+
+		for _, record := range mapToRecords(totals) {
+			recordDateString := record[0]
+			eventTime, err := time.Parse("2006-01-02T00:00:00Z", recordDateString+"T00:00:00Z")
+			if err != nil {
+				log.Fatal("time.Parse: ", err)
+			}
+
+			metrics := map[string]okay.Metric{
+				"totalCount":       okay.Count(totals[recordDateString]),
+				"incidentDuration": okay.Duration(time.Duration(time.Duration(incidents[recordDateString]) * time.Minute)),
```

Comment:
```suggestion
				"incidentDuration": okay.Duration(time.Duration(incidents[recordDateString]) * time.Minute),
```

I'm _think_ this works without double-casting it 🤔